week-1/3-extra/3-magic-8-ball.js

```diff
@@ -42,20 +42,66 @@ My sources say no.
 Outlook not so good.
 Very doubtful.
 */
-
+let veryPositive = [
+    "It is certain.",
+    "It is decidedly so.",
+    "Without a doubt.",
+    "Yes - definitely.",
+    "You may rely on it."
+]
+let positive = [
+    "As I see it, yes.",
+    "Most likely.",
+    "Outlook good.",
+    "Yes.",
+    "Signs point to yes."
+]
+let negative = [ 
+              "Reply hazy, try again.",
+              "Ask again later.",
+              "Better not tell you now.",
+              "Cannot predict now.",
+              "Concentrate and ask again."
+]
+let veryNegative = [
+              "Don't count on it", 
+              "My reply is no", 
+              "My sources say no.", 
+              "Outlook not so good.", 
+              "Very doubtful."
+]
+let posibleTypes = [
+              "very positive",
+              "positive",
+              "negative",
+              "very negative"
+]
+let possibleAnswers = [ veryPositive, positive, negative, veryNegative  ]
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  console.log("The ball has shaken!")
+  let answerType = Math.floor(Math.random() * 4)
```

Comment:
Not sure this works... correct me if I'm wrong
if `answerType === 2`
then `possibleAnswers[ answerType ] => "positive"`
then `selectedAnswer = "positive"`
then `return selectedAnswer[2] => "s" // the third character of the string`

Reply:
I guess I should have made more effort and included comments...

`answerType === 2` refers to the 'negative' answer category which will index in the 3rd entries of the possibleAnswers, and posibleTypes arrays:

`possibleAnswers[ 2 ] => [ "Reply hazy, try again.", "Ask again later.", "Better not tell you now.", "Cannot predict now.", "Concentrate and ask again." ]`

`posibleTypes[ 2 ] => 'negative'`